chapters/glossary.md

```diff
@@ -48,6 +51,16 @@ An `interface` (or, more properly said, `network interface`) is (typically) an e
 
 An internet is a set of networks joined together by routers so that machines can communicate across those networks to one other.
 
+## IP address
+
+An IP (Internet Protocol) address is the layer 3 address of a machine on a network.
+<!-- TODO: write a definition for IP address -->
+
+## MAC address
+
+A MAC (Media Access Control) address is the layer 2 address of a machine on a network. If you'd like to read more about what a MAC address is in technical terms, checkout [Appendix: Understanding tcpdump](../appendix/understanding-tcpdump.md).
```

Comment:
I'm not sure what you're linking to here? And why we would link to a document on tcpdump instead of the one on MAC addresses?